src/hapi/pipelines/database/conflict_event.py

```diff
@@ -46,6 +47,7 @@ def populate(self):
                 values = admin_results["values"]
 
                 for admin_code in admin_codes:
+                    admin_rows = []
```

Comment:
Try making this a set

Reply:
Unfortunately I can't add a dict to a set! Unhashable, so I think I'm stuck with list.

Reply:
Not sure if it's still needed, but you can use a named tuple instead of a dict and then you could add that to a set. Ex:
```python
from collections import namedtuple
# defined named tuple and its fields
MyTuple = namedtuple('MyTuple', ['resource_hdx_id', 'events'])

item = MyTuple(resource_hdx_id='abcd-efgh', events=5)
a_set = set()
a_set.add(item1)

# If you need the item as a dict
item1._asdict() # {'resource_hdx_id': 'abcd-efgh', 'events': 5}
```

Not sure how big `admin_rows` gets. But if it does, having a set can make a difference.

Reply:
I checked and it didn't make any difference when I ran the full set of data (all countries and all years).  Next PR I can switch it to a set and see if it's faster on github.  admin_rows will have a max length of around 100 for most cases, going up to several hundred for a few countries.